src/stories/MultiSelect.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { Story } from '@storybook/react';
+import React, {useState} from 'react';
+import MultiSelect, { MultiSelectProps } from '../components/MultiSelect';
+
+export default {
+  title: 'MultiSelect',
+  component: MultiSelect,
+};
+
+const Template: Story<MultiSelectProps<number>> = (args) => {
+  const [selectedOptions, setSelectedOptions] = useState<number[]>([]);
+
+  const handleAdd = (id: number) => {
+    const newOptions = [...selectedOptions];
+    newOptions.push(id);
+    setSelectedOptions(newOptions);
+  };
+
+  const handleRemove = (id: number) => {
+    const newOptions = [...selectedOptions];
+    const index = newOptions.findIndex((value) => id === value);
+    if (index >= 0) {
+      newOptions.splice(index, 1);
+      setSelectedOptions(newOptions);
+    }
+  };
+
+  return <MultiSelect {...args} selectedOptions={selectedOptions} onAdd={handleAdd} onRemove={handleRemove} />;
+};
+
+export const Default = Template.bind({});
+
+const defaultOptions = new Map<number, string>([
```

Comment:
rename the duplicate spam as something else?

Reply:
but that's... the joke 😅 
(I'll get rid of it since it's confusing)

Reply:
yes i totally got the spam joke :)

Reply:
fixed it 😸